sdktests/common_tests_events_contexts.go

```diff
@@ -169,23 +145,26 @@ func (c CommonEventTests) EventContexts(t *ldtest.T) {
 	dataSource := NewSDKDataSource(t, data)
 
 	for _, p := range makeEventContextTestParams() {
+		outputMatcher := func(context ldcontext.Context) m.Matcher {
+			expectedContext := context
+			if p.outputContext != nil {
+				expectedContext = p.outputContext(context)
+			}
+			return JSONMatchesEventContext(expectedContext, p.redactedShouldBe)
+		}
+
 		t.Run(p.name, func(t *ldtest.T) {
 			contexts := p.contextFactory("doServerSideEventContextTests")
 			events := NewSDKEventSink(t)
-			client := NewSDKClient(t, WithEventsConfig(p.eventsConfig), dataSource, events)
+			client := NewSDKClient(t, c.baseSDKConfigurationPlus(WithEventsConfig(p.eventsConfig), dataSource, events)...)
```

Comment:
Omitting this caused it to initialize the client-side SDK test service without the required initial user parameter.